packages/components/src/List/Toolbar/ColumnChooserButton/service/columnChooser.utils.js

```diff
@@ -0,0 +1,43 @@
+function findMatchingIemLabel(label) {
```

Comment:
```suggestion
function findMatchingItemLabel(label) {
```

Reply:
In fact you can even remove this function, adding 1 LoC in the `columnChooserCollection.find()`, removing 1 level of complexity